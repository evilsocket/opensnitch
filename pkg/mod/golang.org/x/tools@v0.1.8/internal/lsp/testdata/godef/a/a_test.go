package a

import (
	"testing"
)

func TestA(t *testing.T) { //@TestA,godef(TestA, TestA)
}
