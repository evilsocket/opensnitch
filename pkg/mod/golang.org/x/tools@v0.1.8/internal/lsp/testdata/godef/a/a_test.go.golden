-- TestA-definition --
godef/a/a_test.go:7:6-11: defined here as ```go
func TestA(t *testing.T)
```
-- TestA-definition-json --
{
	"span": {
		"uri": "file://godef/a/a_test.go",
		"start": {
			"line": 7,
			"column": 6,
			"offset": 39
		},
		"end": {
			"line": 7,
			"column": 11,
			"offset": 44
		}
	},
	"description": "```go\nfunc TestA(t *testing.T)\n```"
}

-- TestA-hoverdef --
```go
func TestA(t *testing.T)
```
